src/runtime_src/core/edge/user/shim.cpp

```diff
@@ -1231,6 +1231,17 @@ int ZYNQShim::xclLogMsg(xrtLogMsgLevel level, const char* tag,
     else return 0;
   }
 
+  double ZYNQShim::xclGetDeviceClockFreqMHz()
+  {
+    xclDeviceInfo2  deviceInfo ;
+    xclGetDeviceInfo2(&deviceInfo) ;
+    unsigned clockFreq = deviceInfo.mOCLFrequency[0] ;
```

Comment:
unsigned what?  please fix in all files you have changed.
Ironically, I wouldn't have asked if you had used 'auto' :-)